samples/certificate_provisioning/README.rst

```diff
@@ -48,6 +48,60 @@ sample application (i.e. ``samples/certificate_provisioning``) and type:
 
 See `ESP32`_ for details on how to use ESP32 board.
 
+nRF52840 DK + ESP32-WROOM-32
+----------------------------
+
+This subsection documents using nRF52840 DK running Zephyr with offloaded ESP-AT
+WiFi driver and ESP32-WROOM-32 module based board (such as ESP32 DevkitC rev.
+4) running WiFi stack. See `AT Binary Lists`_ for links to ESP-AT binaries and
+details on how to flash ESP-AT image on ESP chip. Flash ESP chip with following
+command:
+
+.. code-block:: console
+
+   esptool.py write_flash --verify 0x0 PATH_TO_ESP_AT/factory/factory_WROOM-32.bin
+
+Connect nRF52840 DK and ESP32-DevKitC V4 (or other ESP32-WROOM-32 based board)
+using wires:
+
++-----------+--------------+
+|nRF52840 DK|ESP32-WROOM-32|
```

Comment:
Should we add WROVER pins while we're at it? Or maybe a separate PR adding it for all the samples makes more sense.

Reply:
I added the WROVER entry to this one and created #430 for the rest. Please review the new RST syntax I used on the table.